core/tests/protractor_utils/SubTopicViewerPage.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the topic and story viewer page, for use
+ * in Protractor tests.
+ */
+
+const { by } = require('protractor');
```

Comment:
I don't think you need this. See [`ClassroomPage.js`](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/ClassroomPage.js) for example

Reply:
Correct. Got auto-imported I think, removed.